app/models/custom_filter_keyword.rb

```diff
@@ -23,6 +23,10 @@ class CustomFilterKeyword < ApplicationRecord
   before_destroy :prepare_cache_invalidation!
   after_commit :invalidate_cache!
 
+  def keyword=(value)
+    super(value&.strip)
```

Comment:
You can use a `before_validation` callback instead of overriding the setter.

Reply:
@Gargron, I was thinking along the same lines untilI I came across the following thread. I also found a similar usage in the repository so tried to stick to the same conventions.

Thread: https://stackoverflow.com/a/11071577/7768631
Reference: https://github.com/mastodon/mastodon/blob/main/app/models/account_migration.rb#L55

Reply:
@Gargron , The ideal place to validate this is indeed a `before_validation`. Let me know if you still prefer the same. I can create a separate PR to make the changes in other places as well, to keep things uniform throughout the codebase.